app/api/puzzle/submit/route.ts

```diff
@@ -87,6 +110,10 @@ export async function POST(req: NextRequest) {
         new GameResult(getPuzzleRating(puzzle), themeRating)
       );
     }
+    
+    if (activePuzzle.isReview) {
+      scaleRatingDelta(oldRating, themeRating);
+    }
```

Comment:
I think this makes sense - if the puzzle was a review, we don't want to boost all those theme ratings unnecessarily